ols/app/models/config.py

```diff
@@ -494,6 +495,162 @@ def validate_yaml(self) -> None:
             v.validate_yaml()
 
 
+class StdioTransportConfig(BaseModel):
+    """Stdio transport configuration for MCP server."""
+
+    command: Optional[str] = None
+    args: Optional[str] = None
+    env: Optional[str] = None
+    cwd: Optional[str] = None
+    encoding: Optional[str] = None
+
+    def __init__(self, data: Optional[dict] = None) -> None:
+        """Initialize configuration and perform basic validation."""
+        super().__init__()
+        if data is None:
+            raise InvalidConfigurationError("stdio transport configuration is missing")
+        if "command" not in data:
+            raise InvalidConfigurationError(
+                "command needs to be specified for STDIO transport"
+            )
+        self.command = data.get("command", "")
+        self.args = data.get("args", "")
+        self.env = data.get("env", constants.STDIO_TRANSPORT_DEFAULT_ENV)
+        self.cwd = data.get("cwd", constants.STDIO_TRANSPORT_DEFAULT_CWD)
+        self.encoding = data.get("command", constants.STDIO_TRANSPORT_DEFAULT_ENCODING)
```

Comment:
is the enconding meant to equal de command?

Reply:
nah

Reply:
fixed @JoaoFula , thank you